src/main/java/pl/craftserve/radiation/LugolsIodinePotion.java

```diff
@@ -237,4 +245,49 @@ public ItemStack getResult(int index) {
             return this.results[index];
         }
     }
+
+    //
+    // Config
+    //
+
+    public interface Config {
+        String name();
+        Duration duration();
+        String drinkMessage();
+    }
+
+    public static class ConfigImpl extends BaseConfig implements Config {
```

Comment:
To samo co wyzej pisalem np, BarConfig. Robisz klasy i inferfejsy w momencie kiedy w ogole tego nie potrzebujesz. Unit testow tez na to nie ma to po co komplikowac kod?

Reply:
Oddzielam strukturę konfiguracji od zależności od konfiguracji Bukkit. Umożliwia to utworzenie obiektów np `Radiation` bez kombinowania z YAML albo `MemoryConfiguration`.